input/fsh/CancerRelatedMedicationResources.fsh

```diff
@@ -0,0 +1,17 @@
+Profile:  CancerRelatedMedicationRequest
+Parent:   USCoreMedicationRequest
+Id:       mcode-cancer-related-medication-request
+Title:    "Cancer-Related Medication Request"
+Description:    "A record of the medication prescription or order for a patient whose condition is related to a primary or secondary cancer condition. The request may be reported by the prescriber, prescribing organization, or patient does not have to be directly observed."
+* ^status = #draft
+* ^experimental = true
+* subject 1..1 MS
+* subject only Reference(CancerPatient)
+* extension contains
+    TreatmentIntent named treatmentIntent 0..1 MS and
+    TerminationReason named terminationReason 0..* MS
+* reasonCode MS
+* reasonReference MS
+* reasonReference only Reference(PrimaryCancerCondition or SecondaryCancerCondition or ComorbidConditionParent) // only for cancer-related reasons
```

Comment:
1) You could simplify this to Reference(CancerConditionParent). 
2) Why include ComorbidConditions? That entails just about anything -- e.g., Metformin for pre-diabetes.

Reply:
Because it could be equally relevant for contraindications to administer an antineoplstic drug.
I'll make the optimizing change to referernce CancerConditionParent.